opta/commands/apply.py

```diff
@@ -358,8 +361,26 @@ def _verify_parent_layer(layer: Layer) -> None:
                 "Please fix these issues and try again!"
             )
     except MissingState as e:
-        raise MissingState(
-            f"Failed to get the Environment state {e.args[0]}"
-            "Usually, this means that the Environment mentioned in configuration file does not exist."
-            "You can read more about this in our getting started guide: https://docs.opta.dev/getting-started/"
+        click.confirm(
+            f"Failed to get the Environment state {e.args[0]} "
+            "Usually, this means that the Environment mentioned in configuration file does not exist. \n"
+            "Would you like to create your environment?",
```

Comment:
It would be nice to also tell the user which environment file will be used (this info should be in the service yaml file). For example the question  could be

"Would you like to create your environment using /home/foo/optafiles/fooenv.yaml?"